src/Zed/Common/Factory/StatelessFactoryRule.php

```diff
@@ -1,52 +0,0 @@
-<?php
-
-namespace ArchitectureSniffer\Zed\Common\Factory;
-
-use PHPMD\AbstractNode;
-use PHPMD\AbstractRule;
-use PHPMD\Node\ClassNode;
-use PHPMD\Rule\ClassAware;
-
-/**
- * Factories shouldn't contain any property. Properties would make the factory stateful.
- */
-class StatelessFactoryRule extends AbstractRule implements ClassAware
```

Comment:
I already did before he left. On this one.
We can add another more sane (with proper explanation) rule in the future here.